packages/hir/src/lowering/packages.candy

```diff
@@ -0,0 +1,22 @@
+use compiler_utils
+use incremental
+
+fun getDirectDependencies(context: QueryContext<List<CompilerError>>, package: Package): Set<Package> {
+  // Returns all direct dependencies of the given `package` as declared in the `.candyspec`.
+
+  todo("getDirectDependencies")
+}
+
+fun getTransitiveDependencies(context: QueryContext<List<CompilerError>>, package: Package): List<Package> {
+  // Returns all transitive dependencies of the given `package` in a topological order.
+  
+  todo("getTransitiveDependencies")
```

Comment:
Similar to `getAllPackagesInCompilationOrder` from `compiler_utils/src/dependencies.candy`, just without the current package included.